dcos/mesos.py

```diff
@@ -442,6 +459,35 @@ def tasks(self, fltr=None, completed=False):
 
         return tasks
 
+    # TODO: Currently waiting on this mapping in state.json to be present.
```

Comment:
We should also be checking for the existence of this feature in `cli/dcoscli/task/main.py`. Is the plan to still use `capabilities` for this?